Assets/Scripts/ConditionSetter.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ConditionSetter : MonoBehaviour
+{
+    public string condition;
+
```

Comment:
Why do you need to have it as a monobehaviour? Wouldn't this all work as a static class or a singleton? Then you wouldn't need to put it to the scene and search for it, but rather just access directly

Reply:
not sure If I would be able to call the function from it in the same way

Reply:
Could you try? :D

Reply:
waste of time, documentation does not describe such a way of sending data